game-core/src/main/java/games/strategy/triplea/ui/MovePanel.java

```diff
@@ -1147,41 +1149,33 @@ private void updateUnitsThatCanMoveOnRoute(final Collection<Unit> units, final R
       best = CollectionUtils.getMatches(best, Matches.unitIsNotSea());
     }
     sortUnitsToMove(best, route);
-    Collections.reverse(best);
-    List<Unit> bestWithDependents = addMustMoveWith(best);
-    final MoveValidationResult allResults;
-    getData().acquireReadLock();
-    try {
-      final MoveDescription move =
-          new MoveDescription(bestWithDependents, route, unitsToTransports, dependentUnits);
-      allResults =
-          AbstractMoveDelegate.validateMove(
-              moveType, move, getCurrentPlayer(), nonCombat, getUndoableMoves(), getData());
-    } finally {
-      getData().releaseReadLock();
-    }
+    best = Lists.reverse(best);
```

Comment:
Not really a fan of re-assigning this variable, because it makes things confusing.
I'd appreciate if you could assign this wrapper to a new independent `bestReversed` variable. (If you want you can even use `Collection#removeIf` with an inverted `Predicate` instead if the repeated `CollectionUtils#getMatches` which avoids having to recreate a list every time and make the variable final)